src/ui/components/Library/Team/View/TestRuns/Overview/TestResultListItem.tsx

```diff
@@ -25,7 +24,7 @@ export function TestResultListItem({
   recording: Recording;
   secondaryBadgeCount: number | null;
 }) {
-  const { comments, metadata } = recording;
+  const { comments, metadata, isProcessed } = recording;
```

Comment:
At some point in the commits yesterday, the `isProcessed` flag got dropped and is now no longer controlling the visual style of these icons, which seems to be the primary purpose of this PR.

Reply:
Fixed here 933cd557b503cde18ee230db640076e6f5fbdc0f